service/docs/source/geopm_pio_sst.7.rst

```diff
@@ -0,0 +1,197 @@
+  by the number of high priority cores (cores in CLOS 0 or 1), and
+  they determine the frequencies that are obtainable by those
+  cores. Generally, if there are fewer high priority cores, the
+  increase in turbo frequency limit is greater.
+
+* ``SST::HIGHPRIORITY_FREQUENCY_SSE:n``:
+  (Package scope) Returns the high priority turbo frequency for bucket
+  n at the SSE license level.
+
+* ``SST::HIGHPRIORITY_FREQUENCY_AVX2:n``:
+  (Package scope) Returns the high priority turbo frequency for bucket
+  n at the AVX2 license level.
+
+* ``SST::HIGHPRIORITY_FREQUENCY_AVX512:n``:
+  (Package scope) Returns the high priority turbo frequency for bucket
+  n at the AVX512 license level.
+
+* ``SST::LOWPRIORITY_FREQUENCY:[SSE|AVX2|AVX512]``:
+  (Package scope) RReturns the low-priority turbo frequency of the
+  specified licence level. Note these frequencies do not change based
+  on the number of high priority cores.
+
+
+CONFIGURATION
+~~~~~~~~~~~~~
+
+* ``SST::TURBO_ENABLE``:
+  (Package scope) Returns 1 if SST-TF feature is enabled, 0 if
+  disabled.
+
+* ``SST::COREPRIORITY_ENABLE``:
+  (Package scope) Returns 1 if SST-CP feature is enabled, 0 if
+  disabled.
+
+* ``SST::COREPRIORITY:ASSOCIATION``:
+  (Core scope) Returns the core's assigned CLOS.
+
+* ``SST::COREPRIORITY:n:WEIGHT``:
+  (Package scope) Returns proportional priority for CLOS n. A lower
+  weight indicates a higher priority. Weight ranges from 0-15 and is
+  used to distribute power amongst cores.
+
+* ``SST::COREPRIORITY:n:FREQUENCY_MIN``:
+  (Package scope) Returns the minimum frequency of CLOS n. Given
+  sufficient power headroom, all cores will receive this minimum
+  frequency before any remaining power is distributed.
+
+* ``SST::COREPRIORITY:n:FREQUENCY_MAX``:
+  (Package scope) Returns the maximum frequency of CLOS n. Power will
+  not be distributed to cores beyond this maximum frequency.
+
+CONTROLS
+--------
+
+* ``SST::TURBO_ENABLE``:
+  (Package scope) Enable SST-TF feature.
+
+* ``SST::COREPRIORITY_ENABLE``:
+  (Package scope) Enable SST-CP feature.
+
+* ``SST::COREPRIORITY:ASSOCIATION``:
+  (Core scope) Assign a core to a CLOS.
+
+* ``SST::COREPRIORITY:n:WEIGHT``:
+  (Package scope) Set proportional priority for CLOS n. A lower weight
+  indicates a higher priority. Weight ranges from 0-15 and is used to
+  distribute power amongst cores.
+
+* ``SST::COREPRIORITY:n:FREQUENCY_MIN``:
+  (Package scope) Set the minimum frequency of CLOS n. Given
+  sufficient power headroom, all cores will receive this minimum
+  frequency before any remaining power is distributed.
+
+* ``SST::COREPRIORITY:n:FREQUENCY_MAX``:
+  (Package scope) Set the maximum frequency of CLOS n. Power will not
+  be distributed to cores beyond this maximum frequency.
+
+EXAMPLE
+-------
+
+The following example uses geopmread and geopmwrite command-line
+tools.  These steps can also be followed within an agent. Enabling
+steps are also in the ``SSTFrequencyGovernor``.
```

Comment:
Probably shouldn't refer to that class in docs yet.

`SSTFrequencyGovernor` isn't targeted until 2.1. I'm also looking into whether I can take care of that in `FrequencyGovernor` instead.